API.md

```diff
@@ -628,4 +628,7 @@ json
 ]
 ```
 
-An `HTTP 404 Not Found` status code is returned if an event could not be found.
\ No newline at end of file
+An `HTTP 404 Not Found` status code is returned if an event could not be found.
+
+## Get Events linked to list of Collection Exercises
+* `GET /collectionexercises/events?ids={a,b}` will return the events linked to the specified collection exercise IDs.
```

Comment:
Can you include an example?  Using this, I would've put the curly braces in but the unit test says otherwise.
Can you also remove the curly braces from this example as they're not needed and are misleading